src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -876,3 +876,8 @@ Prevent Long URL From Breaking Out of Container
         white-space: nowrap;
     }
 }
+
+.toolTipDecorate {
```

Comment:
Let's use `.tool-tip-decorate`, and maybe find a better place in the file to put this rule